test/preconditioner/jacobi_kernels.cpp

```diff
@@ -323,20 +307,23 @@ TEST_F(Jacobi, HipPreconditionerEquivalentToRefWithBlockSize32Unsorted)
 }
 
 
-#if GINKGO_HIP_PLATFORM_HCC
-TEST_F(Jacobi, HipPreconditionerEquivalentToRefWithBlockSize64)
+#ifdef GKO_COMPILING_HIP
```

Comment:
Do we not test hip on CUDA GPUs anymore ?

Reply:
Blocksize 64 is only enabled for ROCm, we check the warp size in the test body